lib/fileutils.rb

```diff
@@ -1485,7 +1491,7 @@ def postorder_traverse
           end
         end
       end
-    ensure
```

Comment:
I realized this `ensure` might have been intentional.

The current behavior is, even if an exception happen, still try to remove the given folder (and most likely get a `Errno::ENOTEMPTY` exception, because the original exception probably prevented some of the files inside to be removed). In that case, the original exception is still reported as the exception cause.


In Bundler, I was not seeing the original error because we're failing to report the cause if it exists in a couple of places, but I will fix this in Bundler.

This is all to say that, if the behavior I proposed is to be implemented, we might still want to restore this `ensure`.